app/lib/client/methods/sendMessage.js

```diff
@@ -19,7 +18,7 @@ Meteor.methods({
 			return toastr.error(t('Message_Already_Sent'));
 		}
 		const user = Meteor.user();
-		message.ts = isNaN(TimeSync.serverOffset()) ? new Date() : new Date(Date.now() + TimeSync.serverOffset());
```

Comment:
Why are we removing this?

Reply:
Because of serverOffset, there was a msg sync issue. Anyway, we are allowing all the msg on the server with `new Date()` unlike before where we were checking the ts, and allowing not more than 60000.
https://github.com/WideChat/Rocket.Chat/blob/7fdf5e6f04437d3792aff5591e7947e64fcec421/app/lib/server/methods/sendMessage.js#L38